db/index.js

```diff
@@ -0,0 +1,7 @@
+const { Pool } = require('pg');
+
+const config = require('../dbConfig');
+
+const pool = new Pool(config);
+
+exports.query = (text, params) => pool.query(text, params);
```

Comment:
Does this need to be in it's own file, or should it hang out with dbConfig?

Reply:
Meh, [pg (aka node-postgres](https://node-postgres.com/) suggests the `db/index.js`... AND looks like (finally? this is great... but needs to be validated) you can use a connection string instead of the connection string parsing nonsense. Suggest update to connection string here and removing dbConfig if all is well in these lands: https://node-postgres.com/features/connecting#Connection%20URI